rl_algorithms/common/networks/base_network.py

```diff
@@ -0,0 +1,30 @@
+import torch
+import torch.nn as nn
+
+from rl_algorithms.dqn.networks import IQNMLP
+
+
+class Base_network(nn.Module):
```

Comment:
`BaseNetwork` is better than the current name.
We use **Pascal Case** for class name. ([reference](https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841))

Reply:
should add simple description for class and each function.

Reply:
resolved the issue in commit 1070159.